py/picca/utils.py

```diff
@@ -470,3 +470,127 @@ def unred(wave, ebv, R_V=3.1, LMC2=False, AVGLMC=False):
     corr = 1. / (10.**(0.4 * curve))
 
     return corr
+
+
+def catalog_from_zbest(zbestfiles,outfile,zmin,zmax): 
```

Comment:
What is the purpose of this function? Inputs? Outputs? This function should have a docstring explaining that. If I understood it properly its purpose is to generate a catalogue from a list of files. If so, wouldn't this function be better placed in converters.py?

Reply:
The idea of the function was to generate a catalog, I can add docstrings and move to converters. Not sure if we still want any of this as there's been some additional work on catalogs in cosmodesi

Reply:
I think we can include it, given that it is already writen. Adding it to `converters.py` will keep the code clean even if we don't need to used it in the future.

Reply:
the catalog making is removed now